src/main/java/sx/blah/discord/util/RequestBuffer.java

```diff
@@ -51,11 +53,21 @@ public static void request(IVoidRequest request) {
 	}
 
 	/**
+	 * This returns the number of incomplete requests.
+	 *
+	 * @return The number of incomplete requests.
+	 */
+	public static int getIncompleteRequestCount() {
+		return incompleteRequestCount;
```

Comment:
You could also just count the entries in the `requests` field. Much simpler and would be more maintainable in the future.